karma.conf.js

```diff
@@ -0,0 +1,55 @@
+module.exports = config => {
+  config.set({
+    basePath: '',
+    frameworks: ['tap'],
+    files: [
+      {pattern: 'test/integration/index.js'},
+    ],
+    preprocessors: {
+      'test/integration/*.js': ['webpack', 'sourcemap'],
+    },
+    webpack: {
+      node: {
+        fs: 'empty',
+      },
+      // Uncomment when debugging
+      // devtool: 'inline-source-map',
+      module: {
+        rules: [{
+          enforce: 'post',
+          test: /\.js$/,
+          exclude: [
+            /(test|node_modules)\//,
+            'src/p5.dance.interpreted.js',
+          ],
+          loader: 'istanbul-instrumenter-loader'
+        }]
+      }
+    },
+    reporters: ['tap-pretty', 'coverage'],
+
+    // Run headless unless WATCH=1.
+    browsers: process.env['WATCH'] ? ['Chrome'] : ['ChromeHeadless', 'ChromeHeadlessNoSandbox'],
+
+    customLaunchers: {
+      ChromeHeadlessNoSandbox: {
+        base: 'ChromeHeadless',
+        flags: ['--no-sandbox']
```

Comment:
What is this needed for?

Reply:
This is from the Travis guide on how to use chrome headless in their container environment.

Same with the dist and addon change. (We may not need them, but without the addon we run on chrome 62 I believe). Just counting on whatever happens to be preinstalled

Reply:
This is from the Travis guide on how to use chrome headless in their container environment.

Same with the dist and addon change. (We may not need them, but without the addon we run on chrome 62 I believe). Just counting on whatever happens to be preinstalled

Reply:
Looks like it is not needed because we're not in the container based environment.

I was following this: https://docs.travis-ci.com/user/chrome

But then this is more important: https://blog.travis-ci.com/2018-10-04-combining-linux-infrastructures